react/src/components/atoms/buttons/ButtonWithIcon/style.scss

```
@@ -1,10 +1,54 @@
 @import 'global';
+@import '00-base/mixins';
 @import "01-atoms/button-search";
 
+$colors: (
+  "c-highlight": $c-highlight,
+  "c-primary": $c-primary,
+  "c-primary-alt": $c-primary-alt,
+  "c-black": $c-black,
+  "c-gray-dark": $c-gray-dark,
+  "c-white": $c-white
+);
+@mixin ma-button-icon-secondary ($color: $c-primary-alt) {
+  svg {
+    fill: $color;
+  }
+  &:hover {
+    svg {
+      fill: $c-white;
+    }
+  }
+}
+@mixin ma-button-icon-quaternary ($color: $c-primary-alt) {
+  border-bottom-color: rgba($color,0.75);
+  color: $color;
+  svg {
+    fill: rgba($color,0.75);
+  }
+  &:hover {
+    border-bottom-color: $color;
+  }
+}
+@mixin ma-button-icon-quaternary-simple ($color: $c-primary-alt) {
```

can we set the default to $c-primary instead
```suggestion
@mixin ma-button-icon-quaternary-simple ($color: $c-primary) {
```

---

Please see comment above, do to this I cannot change the default theme away from primary-alt